eras/alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-alonzo
-version:            1.10.2.1
+version:            1.10.2.0
```

Comment:
Why are you downgrading the version to the one that was released?
```suggestion
version:            1.10.2.1
```

Reply:
Thanks for catching this. I'm not sure why I'm having such a hard time getting this right.